### PR TITLE
feat: Connected Feed Component

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed?

This code is now rendering the HOME_TAB in this `templates` branch
https://github.com/ApollosProject/apollos-templates/pull/196
There's also a story that you can look at that shows the data flowing through the new schema.  

(loading state looks like wack because we have pretty bad in rock) 

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-data-connector-rock/src/features/resolver.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { get } from 'lodash';
 import { createGlobalId } from '@apollosproject/server-core';
+import ApollosConfig from '@apollosproject/config';
```
Comment (vinnyjth) [inline packages/apollos-data-connector-rock/src/features/resolver.js:3]:
```suggestion
```

File: packages/apollos-ui-connected/src/HorizontalCardListFeatureConnected/HorizontalCardListFeatureConnected.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const HorizontalCardListFeatureConnected = ({
     {({ data, loading, refetch }) => {
       if (featureId && refetch && refetchRef)
         refetchRef({ refetch, id: featureId });
-
+      console.log(data);
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/HorizontalCardListFeatureConnected/HorizontalCardListFeatureConnected.js:20]:
```suggestion
```

File: packages/apollos-ui-kit/src/FeedView/index.js [hunk 3]
```diff
@@ -87,11 +87,9 @@ class FeedView extends Component {
         data={content}
         keyExtractor={keyExtractor}
         ListEmptyComponent={
-          error && !isLoading && (!content || !content.length) ? (
-            <ErrorCard error={error} />
-          ) : (
-            ListEmptyComponent
-          )
+          error && !isLoading && (!content || !content.length)
+            ? console.warn(error) || <ErrorCard error={error} />
```
Comment (vinnyjth) [inline packages/apollos-ui-kit/src/FeedView/index.js:91]: I intend to keep this. It's nice to be able to see _why_ an error card was rendered while developing.

File: packages/apollos-ui-connected/src/FeaturesFeedConnected/FeaturesFeedConnected.stories.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      "Celebrate your freedom in Christ — in the car, shower, cubicle or wherever you go — with this playlist of NewSpring Worship's favorite songs of the summer.",
+    coverImage: {
+      sources: [
+        {
+          uri: 'https://picsum.photos/800',
+        },
+      ],
+    },
+    __typename: 'CardListItem',
+    id: 'CardListItem:789',
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/FeaturesFeedConnected/FeaturesFeedConnected.stories.js:26]:
```suggestion
    id: 'CardListItem:7891',
```

File: packages/apollos-ui-connected/src/FeaturesFeedConnected/index.js [hunk 5]
```diff
@@ -1,6 +1,13 @@
...
+} from './FeaturesFeedConnected';
+import GET_FEATURE_FEED from './getFeatureFeed';
 
 export featuresFeedComponentMapper from './featuresFeedComponentMapper';
-export GET_FEED_FEATURES from './getFeedFeatures';
 
+export {
+  FEATURE_FEED_ACTION_MAP,
+  GET_FEATURE_FEED,
+  GET_FEATURE_FEED as GET_FEED_FEATURES,
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/FeaturesFeedConnected/index.js:11]:
```suggestion
  // TODO: deprecated, warn on import and remove later
  GET_FEATURE_FEED as GET_FEED_FEATURES,
```

**Review by redreceipt — APPROVED** [review]

**Pull request merged**